h/util/document_claims.py

```diff
@@ -14,6 +14,19 @@
 
 from __future__ import unicode_literals
 
+import re
+
+
+# Pattern that matches DOIs (eg. "10.1000/123456").
+#
+# See https://www.doi.org/doi_handbook/2_Numbering.html#2.2 The registrant code
+# is not specified as numeric there but is currently always a 4+ digit value.
+# See https://www.doi.org/overview/DOI_article_ELIS3.pdf.
+#
+# This also accepts DOIs represented as URLs (eg.
+# "https://doi.org/10.1000/123456").
```

Comment:
Elsewhere in h we seem to use the `#:` comment style for docstrings of constants and things like this. (autodoc understands it)